src/graph.rs

```diff
@@ -211,9 +211,22 @@ impl StackGraph {
     pub fn get_or_create_file<S: AsRef<str> + ?Sized>(&mut self, name: &S) -> Handle<File> {
         self.add_file(name).unwrap_or_else(|handle| handle)
     }
+
+    /// Returns the file with a particular name.  Panics if there is no file with the requested
+    /// name.
+    pub fn get_file_unchecked<S: AsRef<str> + ?Sized>(&self, name: &S) -> Handle<File> {
+        let name = name.as_ref();
+        self.file_handles.get(name).copied().unwrap()
```

Comment:
I changed the `unwrap` to `expect` but left out the `format!` part, since that would heap-allocate and construct a new `String` every time this method is called.